samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/CommonMessages_ca.properties

```diff
@@ -36,3 +36,21 @@ action_save_pair=Desa la parella
 delete=Elimina
 delete_attempt=Elimina l\u2019intent
 confirm_delete_attempt=Esteu segur de voler eliminar aquest intent?
+
+# S2U-14
```

Comment:
@mpellicer and @bgarciaentornos, could you help me with the CA translations as well please?

Reply:
Sure, I'll help in a bit.